pages/contribute.js

```diff
@@ -99,6 +99,8 @@ class TiersPage extends React.Component {
     const waysToContribute = [];
     const canContribute = collective.isActive && collective.host;
     const hasContributors = this.hasContributors(collective.contributors, collective.events);
+    const hasEventContributors = this.hasContributors([], collective.events);
+    const hasProjectContributors = this.hasContributors([], collective.projects);
```

Comment:
We're a bit hacking the prototype of `hasContributors` here. Could we find a better function signature to mak it a single call?

I can suggest: `hasContributors(collective, verb)`.

The other reason for that is that the function uses `memoizeOne`, by calling it 3 times the current approach breaks memoization.